api/controllers/subdir/actions2.js

```diff
@@ -44,22 +49,30 @@ module.exports = {
 
   exits: {
     success: {
-      description: 'Another success',
-      outputExample: 'Some dynamic message like this.'
+      description: 'A successful result',
+    },
+    alternateSuccess: {
+      description: 'Alternate success',
+      outputExample: 'Some dynamic message like this.',
+    },
+    alternateSuccess2: {
+      description: 'Alternate success (2)',
+      outputExample: 93.45,
```

Comment:
I know this is not related to this PR, but kinda weird that machine as action paradigm uses `example` for input and `outputExample` for exits. Maybe there's a reason for not being consistent and use `example`

Reply:
Yes, there is some inconsistency.